inference-engine/ie_bridges/java/README.md

```diff
@@ -0,0 +1,21 @@
+## Software Requirements 
+- openjdk 11
+
+### Linux
+To install openjdk use: 
+```bash
+sudo apt-get install -y openjdk-11-jdk
+```
+
+## Building on Linux
+
+Build Inference Engine Java API alongside with the Inference Engine build. 
+You need to run Inference Engine build with the following flags:
+
+```bash
+  cd <IE_ROOT>
+  mkdir -p build
+  cd build
+  cmake -DENABLE_JAVA=ON ..
+  make --jobs=$(nproc --all)
```

Comment:
Better not to interfere with generic OpenVINO install step. Just sugget used to add `-DENABLE_JAVA=ON` option to `cmake` command. See Python guide for example